pallets/investments/src/lib.rs

```diff
@@ -826,9 +830,10 @@ where
 				},
 			);
 
-			let collected_redemption = CollectedAmount {
+			let collected_redemption = CollectedRedemption {
 				amount_collected: collection.payout_investment_redeem,
 				amount_payment,
+				amount_remaining_collectable: collection.remaining_investment_redeem,
```

Comment:
This is not the remaining amount to be collectable. This is the `unfulfilled` amount. i.e. the amount of tranche tokens by the still locked for redemption, but not yet swapped into `pool_currency` by the pool.

Reply:
You can never know what is `remaning` as this would mean you already iterate over the fulfilled epochs, which is what we need to prevent for weight reasons.